src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.cs

```diff
@@ -561,15 +616,14 @@ private int GetTokenIndexInStream(SyntaxToken token)
             }
         }
 
-        private sealed class TokenOrderComparer : IComparer<SyntaxToken>
+        private struct TokenOrderComparer : IComparer<SyntaxToken>
```

Comment:
this will cause boxing when this is passed into _tokens.BinarySearch.  Is there a reason this was done?

Reply:
Good point. Reverted.
BCL could have had an overload, that would avoid boxing.